datafusion/core/tests/parquet/page_pruning.rs

```diff
@@ -515,6 +515,120 @@ int_tests!(16);
 int_tests!(32);
 int_tests!(64);
 
+macro_rules! uint_tests {
+    ($bits:expr) => {
+        paste::item! {
+            #[tokio::test]
+            //                      null count  min                                       max
+            // page-0                         0  0                                         4
```

Comment:
I was confused at first, but the `make_uint_batches(0, 5)` returns `0..5` which has a max of 4. ✅